evaluate.py

```diff
@@ -163,18 +163,18 @@ def main(args: argparse.Namespace) -> None:
     if issubclass(TASK, ClassificationTask):
```

Comment:
Need to undo chmod changes

Reply:
sorry, I don't understand what you mean here.

Reply:
This file was previously executable (you could run `./evaluate.py` on Unix) with mode 100755, but is no longer executable with mode 100644. `chmod +x evaluate.py` should do the trick. If you're on Windows and don't know what any of this means, let me know and I can do it.